src/passwordcheck.js

```diff
@@ -28,7 +28,7 @@ define(['bloomfilter',
         PASSWORD_TOO_SHORT: 'PASSWORD_TOO_SHORT'
       };
 
-      function strengthCheck(password) {
+      function isStrongEnough(password) {
         // Check for passwords that at least contain a number and an alphabet,
         // or if alphabets, then at least (minLength + 4) characters long
         var regexString = '((?=.*[a-zA-Z])(?=.*[0-9])[A-Za-z0-9!@#$%^&*()_+ ]{' + minLength + ',' + (minLength + 4) + '})|([A-Za-z0-9!@#$%^&*()_+ ]{' + (minLength + 4) + ',})';
```

Comment:
@shane-tomlinson I believe that would be a good change, but we should also change the name of the function to better reflect that, maybe 'isWeakPassword'? 'Cause, right now we are whitelisting, the change would be checking for weak ones only, is that right?